addon/components/o-s-s/button.stories.js

```diff
@@ -99,13 +100,15 @@ export default {
     },
     countDown: {
       description:
-        "Definition of countDown object, it takes 2 keys : 'callback' (mandatory) - function to call at the end - and 'time' " +
-        "(optional) - time between execute callback -. Key 'time' is representing entire second in millisecond, for exemple 1000, 2000 or 5000",
+        'Definition of countDown object, it takes 3 keys:<br/>' +
```

Comment:
I'm wondering if using backquotes ` would keep formatting and allow us to not use the <br> tags ?

Reply:
Where do you put the backquotes? Instead of single quote?

Reply:
I meant something like :
```typescript
`Definition of countDown object, it takes 3 keys:
- 'callback' (mandatory): function to call at the end
- 'time' (optional): time between execute callback. It is representing entire second in millisecond, for exemple 1000, 2000 or 5000
- 'step' (optional): the step value, it should be in the same unit as the time`
```